fecgov/openFEC
# [DO NOT MERGE!!!]Feature/remove two year restriction

Summary (required)
WIP, NO MERGE!!! this is a testing branch for performance testing of 2-year restriction removal

Resolves # [3595]
Include a summary of proposed changes.
two_year_transaction_period is not a required argument - this allows us to test transaction data beyond 2-year restriction.
(based on recent discussion, I'll modify further to allow for multiple two_year_transaction_periods to take advantage of query performance boost by schedule_a and schedule_b table partitioning. )

How to test the changes locally
check out this branch
point you DB to staging server
run openFEC locally:
./manage.py runserver
test schedule_a and schedule_b endpoints(mainly on response time) with multiple cycles and/or multiple committees.
run rm2year_restriction_test.py against local server:
python rm2year_restriction_test.py --server http://127.0.0.1:5000
all the endpoints should have a valid response coming back within seconds.



-

## Impacted areas of the application
List general components of the application that this PR will affect:

-  



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
